code/modules/mob/living/carbon/human/diona_gestalt.dm

```diff
@@ -275,6 +275,7 @@
 	verbs += /mob/living/carbon/human/proc/switch_to_nymph
 	M.detached = TRUE
 	M.update_verbs(TRUE)
+	M.sync_languages(src)
```

Comment:
This might not be sufficient as sync languages only adds languages that done exist, but does not remove languages the host (src) does not have

Reply:
It actually does. It sets the `src`'s languages to be the disjunction between `src` and `M` languages, and then copies the `src`'s languages to `M`. Technically a new nymph shouldn't know any languages the gestalt doesn't know, so this _should_ be fine? But what I don't get is. Why don't we just do: `M.languages = languages.Copy()`

Reply:
The other point is, whether or not it's a good idea to copy languages like this in the first place, when we have add and remove language procs. :thinking:

Reply:
```suggestion
	M.languages = languages.Copy()
```